teku/src/main/java/tech/pegasys/teku/cli/subcommand/VoluntaryExitCommand.java

```diff
@@ -98,10 +84,33 @@
 
   @Override
   public void run() {
+    SUB_COMMAND_LOG.display("Loading configuration...");
     initialise();
+    confirmExits();
     getValidatorIndices(blsPublicKeyValidatorMap).forEach(this::submitExitForValidator);
   }
 
+  private void confirmExits() {
+    SUB_COMMAND_LOG.display("Exits are going to be generated for validators: ");
+    SUB_COMMAND_LOG.display(getValidatorAbbreviatedKeys());
+    SUB_COMMAND_LOG.display(
+        "\nThese validators won't be able to be re-activated, and withdrawals aren't likely to be possible until Phase 2 of eth2 Mainnet.");
```

Comment:
```suggestion
        "\nThese validators won't be able to be re-activated, and withdrawals are unlikely to be possible until Phase 2 of eth2 Mainnet.");
```

Reply:
Also, can't hard code `\n` - should separate calls to display for each line.

Reply:
that was for an extra line, i could put in a blank display call?

Reply:
Yeah a blank call would work.  `\n` isn't the line separator on all systems.